ratis-netty/src/main/java/org/apache/ratis/netty/server/NettyServerStreamRpc.java

```diff
@@ -91,6 +106,11 @@ private void writeTo(ByteBuf buf, DataStream stream, boolean released) {
           throw new CompletionException(t);
         }
       }
+
```

Comment:
I think we can do `channel.write(buffer);` and `streamOutput.streamAsync(buf.nioBuffer());` in parallel, `channel.write(buffer)`  maybe slow when write file. But not sure, just a discussion.

Reply:
Agreed. Let me think about this. 

Make the writes happen in parallel does make sense.

Reply:
Have changed to make all writes happen in parallel, and have a `allOf` to waits all writes finish before send reply.